src/main/java/android/util/Base64.java

```diff
@@ -0,0 +1,741 @@
+/*
+ * Copyright (C) 2010 The Android Open Source Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package android.util;
```

Comment:
I don't think that it's good idea to use package 'android' in java project. If I'm understand correctly it's Base64 class implementation from android sdk, but please add it to normal package like 'svarzee.gps.gpsoauth.util'
You can also add here implementation from Java8 instead of Android's.